src/orchestrator/registry.ts

```diff
@@ -19,385 +12,100 @@ import {
   sepolia,
   zksync,
 } from 'viem/chains'
+import type { TokenSymbol } from '../types'
+import registryData from './registry.json'
+import type { TokenConfig } from './types'
```

Comment:
Most of the changes here are due to switching to the JSON registry as the source of truth.